packages/admin-panel-server/src/routes/ImportDashboardVisualisationRoute.ts

```diff
@@ -38,25 +73,36 @@ export class ImportDashboardVisualisationRoute extends Route<ImportDashboardVisu
       throw new UploadError();
     }
 
-    const visualisation = readJsonFile<Record<string, unknown>>(this.req.file.path);
-    fs.unlinkSync(this.req.file.path);
+    const { dashboards, dashboardRelations, ...visualisation } = this.readFileContents();
 
     const extractor = new DashboardVisualisationExtractor(
       visualisation,
       draftDashboardItemValidator,
       draftReportValidator,
     );
-    const body = extractor.getDashboardVisualisationResource();
+    const extractedViz = extractor.getDashboardVisualisationResource();
```

Comment:
I am totally with you here... I generally sticked to `visualisation` for consistency, since it was already used in most cases when I jumped into the project, but I would prefer if we used `viz` exclusively and introduce it as a term in our codebase.

Not sure if it's worth refactoring now... But I would like us to promote the idea that using well thought after and established abbreviations makes code more readable! How would our world be if we had to write `findByIdentifier`, `dashboardIdentifier` instead of `findById`, `dashboardId`... 😄

Reply:
Definitely agree with this too, and the extra justification is that `viz` is also what the business (i.e. Erin) uses, so it's not just some software dev abbreviation but still fits with domain driven design :-)

Reply:
Domain driven design ftw!!